services/kinesis/src/it/java/software/amazon/awssdk/services/kinesis/KinesisIntegrationTests.java

```diff
@@ -133,13 +124,9 @@ public void testKinesisOperations() throws Exception {
             Assert.assertEquals(1, shards.size());
             Shard shard = shards.get(0);
 
-            // Just to be really sure in case of eventual consistency...
-            Thread.sleep(5000);
-
-            testPuts(streamName, shard);
-
-            // Wait a bit to make sure the records propagate.
-            Thread.sleep(5000);
+            System.out.println("Putting two records...");
```

Comment:
nit: remove println

Reply:
There's a lot of printlns. Should I remove them all?

Reply:
Ah that would be great!

Reply:
Done